### PR TITLE
[FIX]Fixed the issue of move blog content, when user clicks on + sign se...

### PR DESCRIPTION
**Pull request by msh-odoo**
...cond time and popover is already open then move the content back to original position

Task: https://www.odoo.com/web?#id=12207&view_type=form&model=project.task&menu_id=3942&action=327
Pad: https://pad.odoo.com/p/r.7BeSK5dhVDXhyKVh

### CODE REVIEW

File: addons/website_blog/static/src/js/website_blog.inline.discussion.js [hunk 1]
```diff
@@ -36,12 +36,13 @@ var InlineDiscussion = core.Class.extend({
         self.discussions_handler(self.settings.content);
 
         // Hide the discussion.
-        $('html').click(function(event) {
-            if($(event.target).parents('#discussions_wrapper, .main-discussion-link-wrp').length === 0) {
+        $('html').click(function (event) {
+            if ($(event.target).parents('#discussions_wrapper, .main-discussion-link-wrp').length === 0) {
```
Comment (Gorash) [inline addons/website_blog/static/src/js/website_blog.inline.discussion.js]: Please remove the typo change from the diff (don't add/remove spaces if you don't change the line)

File: addons/website_blog/static/src/js/website_blog.inline.discussion.js [hunk 2]
```diff
@@ -36,12 +36,13 @@ var InlineDiscussion = core.Class.extend({
...
                 self.hide_discussion();
             }
-            if(!$(event.target).hasClass('discussion-link') && !$(event.target).parents('.popover').length){
-                if($('.move_discuss').length){
+            if ((!$(event.target).hasClass('discussion-link') || ($(event.target).hasClass('discussion-link') && $(event.target).hasClass('active'))) && !$(event.target).parents('.popover').length) {
```
Comment (Gorash) [inline addons/website_blog/static/src/js/website_blog.inline.discussion.js]: if ((!$(event.target).hasClass('discussion-link') || ($(event.target).is('.discussion-link.active'))) && !$(event.target).closest('.popover').length) {

Use 'is', it's shorter
Use closest to get the first parent who match (and not get all parents and filter)

thanks

File: addons/website_blog/static/src/js/website_blog.inline.discussion.js [hunk 3]
```diff
@@ -50,6 +51,21 @@ var InlineDiscussion = core.Class.extend({
                         'marginLeft': "+=250px"
                     });
                 }
+            } else if ($(event.target).hasClass('discussion-link')) {
+                if (!$('.move_discuss').length) {
+                    $('[enable_chatter_discuss=True]').addClass('move_discuss');
+                    $('[enable_chatter_discuss=True]').animate({
```
Comment (Gorash) [inline addons/website_blog/static/src/js/website_blog.inline.discussion.js]: Put addClass and animate in one line

File: addons/website_blog/static/src/js/website_blog.inline.discussion.js [hunk 4]
```diff
@@ -50,6 +51,21 @@ var InlineDiscussion = core.Class.extend({
                         'marginLeft': "+=250px"
                     });
                 }
+            } else if ($(event.target).hasClass('discussion-link')) {
```
Comment (Gorash) [inline addons/website_blog/static/src/js/website_blog.inline.discussion.js]: please concatenate the two conditions, the content is the same (the same except: self.hide_discussion() but self.hide_discussion() is call latter with the check on active class name)

Comment (msh-odoo) [inline addons/website_blog/static/src/js/website_blog.inline.discussion.js]: @Gorash  which two conditions ?, self.hide_discussion() resides inside else if ($(event.target).hasClass('discussion-link')), would you please elaborate more, which two condition you want to concatenate.

Comment (msh-odoo) [inline addons/website_blog/static/src/js/website_blog.inline.discussion.js]: @Gorash I got your point

Comment (msh-odoo) [inline addons/website_blog/static/src/js/website_blog.inline.discussion.js]: @Gorash If you are asking to concatenate else if condition in to first if stamtent then note that, first we add some marginLeft and in else if we are removing that marginLeft

Comment (Gorash) [inline addons/website_blog/static/src/js/website_blog.inline.discussion.js]: @msh-odoo I commit a patch to rebase

**Comment by Gorash** [comment]
I have commit a patch (shorter and speeder code) to rebase then @tde-banana-odoo you can merged
(todo: change margin animation by less animation (and use only a toggleClass) for designer)

**Pull request merged**